backend/.env.example

```
@@ -1,4 +1,7 @@
 SAFEQUERY_APP_NAME=SafeQuery API
 SAFEQUERY_ENVIRONMENT=development
-SAFEQUERY_DATABASE_URL=postgresql://safequery:safequery@127.0.0.1:5432/safequery
+SAFEQUERY_APP_POSTGRES_URL=postgresql://safequery:safequery@127.0.0.1:5432/safequery
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SAFEQUERY_APP_POSTGRES_URL key should go before the SAFEQUERY_ENVIRONMENT key

(UnorderedKey)